src/posix.mak

```diff
@@ -125,6 +126,11 @@ CFLAGS := $(WARNINGS) \
 	-fno-exceptions -fno-rtti \
 	-D__pascal= -DMARS=1 -DTARGET_$(OS_UPCASE)=1 -DDM_TARGET_CPU_$(TARGET_CPU)=1 \
 	$(MODEL_FLAG)
+# GCC Specific
+ifeq ($(HOST_CC), g++)
+CFLAGS := $(CFLAGS) \
+    -std=gnu++98
+endif
```

Comment:
It should be possible. My normal workflow is normally:
checkout -b mybranch
# some work and commits
checkout master
pull upstream master
checkout mybranch
rebase master
push -f origin mybranch